src/database/initialization/source_to_json.sql

```diff
@@ -22,16 +23,23 @@ BEGIN
   FROM changemakers
   WHERE changemakers.id = source.changemaker_id;
 
-  RETURN jsonb_strip_nulls(jsonb_build_object(
-    'id', source.id,
-    'label', source.label,
-    'dataProviderShortCode', source.data_provider_short_code,
-    'dataProvider', data_provider_json,
-    'funderShortCode', source.funder_short_code,
-    'funder', funder_json,
-    'changemakerId', source.changemaker_id,
-    'changemaker', changemaker_json,
-    'createdAt', source.created_at
-  ));
+  -- The reason to go through the following hassle is we want null organization IDs returned.
+  -- At the same time, we want to omit non-existent changemakers, funders, and data providers.
+  SELECT jsonb_object_agg(name, value)
+  INTO source_json
+  FROM (VALUES
```

Comment:
This is a clever solution, @bickelj!

It took me a minute to understand, because it's depending on changes elsewhere: the other `_to_json.sql` files have new nullable `keycloakOrganizationId` entries, which were getting stripped out by `jsonb_strip_nulls`. Filtering out the top level without filtering out the nested `NULL`s is a good idea.

I spent a few minutes trying to figure out an alternative solution to see how it compared; I tried merging two JSONB objects with `||`, or leveraging `coalesce()`, and neither worked without `jsonb_set_lax`, which is not nearly as elegant as this!

Reply:
@jasonaowen I wish I could take credit for it, but it was adapted from some SO post. I probably should include a [link to it](https://stackoverflow.com/a/58310353) there in the comments.

Reply:
I added the link.